bottery/cli.py

```diff
@@ -30,7 +31,11 @@ def cli():
 @cli.command('startproject')
 @click.argument('name')
 def startproject(name):
-    # Must validate projects name before creating its folder
+    # name validation - only lowercase letters and underscores
+    if not re.match("^[a-z_]*$", name):
+        print('Project name must contain only lowercase letters and underscores.')
+        exit(-1)
```

Comment:
We should raise an exception here and `Click` is responsible to exit the cli: http://click.pocoo.org/5/exceptions/#which-exceptions-exist

I would go with `BadParameter`.